### DISCUSSION TITLE
Starstruck achievement not showing

### DISCUSSION THREAD

**Discussion by intldds** [community/community#108265] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys,

I have got more than 16 stars in this [repo](https://github.com/intldds/ddd-domain-driven-design) (currently 44) but the Starstruck achievement is not showing in my profile. I've already followed the steps in similar discussions but nothing worked.

**Comment by VerzatileDev** [comment]
It might be holdup on the githubs end, it took me a month for some Achievements to show up.

Though make sure its not hidden on your profile, as you can disable them and enable them to show up.  

Steps
---

1. Open your profile
2. Click on " Achivements "
3. You will now see a panel of all the achivements you have 
4. click on the ones you dont see on your profile
5. click the eye icon

Reference to my own:

![image](https://github.com/community/community/assets/47317248/d88498fd-0117-4d08-804f-c0aa0c19d1dc)

**Comment by queenofcorgis** [comment]
Hi @intldds . 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
